packages/service-library/src/servicelib/long_running_tasks/_redis_store.py

```diff
@@ -23,17 +23,21 @@ def _load_from_redis_hash(data: dict[str, str]) -> dict[str, Any]:
     return {k: json_loads(v) for k, v in data.items()}
 
 
+def to_redis_namespace(namespace: LRTNamespace) -> str:
+    return namespace.upper()
+
+
 class RedisStore:
     def __init__(self, redis_settings: RedisSettings, namespace: LRTNamespace):
         self.redis_settings = redis_settings
-        self.namespace: LRTNamespace = namespace.upper()
+        self.redis_namespace: str = to_redis_namespace(namespace)
 
         self._client: RedisClientSDK | None = None
 
     async def setup(self) -> None:
         self._client = RedisClientSDK(
             self.redis_settings.build_redis_dsn(RedisDatabase.LONG_RUNNING_TASKS),
-            client_name=f"long_running_tasks_store_{self.namespace}",
+            client_name=f"long_running_tasks_store_{self.redis_namespace}",
```

Comment:
NOTE: sop this client name would be something like `long_running_task_store_SOME_REDIS_NAMESPACE`, right?